src/analyzer/analyzerwaveform.cpp

```diff
@@ -12,6 +12,10 @@ namespace {
 
 mixxx::Logger kLogger("AnalyzerWaveform");
 
+constexpr double LOW_MID_FREQ_HZ = 600.0;
+
+constexpr double MID_HIGH_FREQ_HZ = 4000.0;
```

Comment:
```suggestion
constexpr double kLowMidFreqHz = 600.0;

constexpr double kMidHighFreqHz = 4000.0;
```

Reply:
I was just procrastinating while doing Rust development in parallel :rofl: